src/core/Akka/Actor/ActorPath.cs

```diff
@@ -287,6 +286,7 @@ public static bool TryParse(string path, out ActorPath actorPath)
             return true;
         }
 
+        [Obsolete]
```

Comment:
I removed the only usage of this method and I felt like it didn't belong there. There is a `Parse` method in the `Address` class.